_studio/shared/include/mfx_config.h

```diff
@@ -164,4 +160,12 @@
 	#define MFX_ENABLE_FOURCC_RGB565
 #endif
 
+#if defined(MFX_ENABLE_VP9_VIDEO_DECODE)
```

Comment:
move this change higher right after `#if defined(MFX_ENABLE_H265_VIDEO_ENCODE)`, i.e. place similar things together